nodewatcher/modules/frontend/ip_space/templates/network/ip_space.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+
+{% contextblock %}
+    {% load i18n sekizai_tags html_tags url_tags partial_tags leaflet_tags %}
+
+    {% add_data "css_data" "css/ipspace.css" %}
+    {% add_data "js_data" "js/d3.v4.js" %}
+    {% add_data "js_data" "js/point.js" %}
+    {% add_data "js_data" "js/hilbert2d.js" %}
+    {% add_data "js_data_end" "js/jquery.tipsy.js" %}
+    {% add_data "js_data_end" "js/DrawCidrs.js" %}
+
+
+    {% trans "Ip space" as title %}
+{% endcontextblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-2">
+            <div class="dropdown">
+                <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Zoom to subnet<span class="caret"></span></button>
+                <ul class="dropdown-menu" id="topnodes">
+                    <li id="cidr"><a href="#" value="10.0.0.0/8">Loading data</option>
+                </ul>
+            </div>
+        </div>
+        <div class="col-md-8" id="wraper">
+            <svg id="ipspace" class="silky-smooth"></svg>
+        </div>
+    </div>
+    <script>
+    $(document).ready(function(){
```

Comment:
Can you put this into a separate Javascript file and not inline it? You can check how it is done in other modules.s